features/performance/assets/mc-pre-boot-worker-tuning.yaml.in

```diff
@@ -5,8 +5,8 @@ metadata:
         machineconfiguration.openshift.io/role: worker-rt
     name: 11-rt-kernel-worker-rt
 spec:
-    kernelArguments:
-        - 'non_iso_cpumask=00000005'
+    #kernelArguments:
+    #    - 'non_iso_cpumask=00000005'
```

Comment:
OK, but why?

Reply:
I mean why we need to comment these lines out now? Please explain in the comments and/or in the commit message